alchemy/package.json

```diff
@@ -204,17 +184,63 @@
     "@cloudflare/workers-types": "catalog:",
     "@libsql/client": "^0.15.9",
     "@octokit/rest": "^21.1.1",
-    "arktype": "^2.0.0",
-    "cloudflare": "^4.0.0",
     "drizzle-orm": "^0.44.2",
-    "hono": "^4.0.0",
     "prettier": "^3.0.0",
     "stripe": "^17.0.0"
   },
+  "peerDependenciesMeta": {
+    "@aws-sdk/client-dynamodb": {
+      "optional": true
+    },
+    "@aws-sdk/client-iam": {
+      "optional": true
+    },
+    "@aws-sdk/client-lambda": {
+      "optional": true
+    },
+    "@aws-sdk/client-s3": {
+      "optional": true
+    },
+    "@aws-sdk/client-sesv2": {
+      "optional": true
+    },
+    "@aws-sdk/client-sqs": {
+      "optional": true
+    },
+    "@aws-sdk/client-ssm": {
+      "optional": true
+    },
+    "@aws-sdk/client-sts": {
+      "optional": true
+    },
+    "@octokit/rest": {
+      "optional": true
+    },
+    "@libsql/client": {
+      "optional": true
+    },
+    "drizzle-orm": {
+      "optional": true
+    },
+    "prettier": {
+      "optional": true
+    },
+    "stripe": {
+      "optional": true
+    }
+  },
   "devDependencies": {
     "@asteasolutions/zod-to-openapi": "^7.3.0",
+    "@aws-sdk/client-ec2": "^3.868.0",
```

Comment:
If the peer dependency is marked as optional, then it won't be installed in our repo unless it's also here as a dev dependency, hence the duplication

Reply:
consistently use `^3` or at least align all of the client-* packages to the same version